src/main/resources/assets/minebound/lang/en_us.json

```diff
@@ -76,17 +76,33 @@
   "item.minebound.magic_telekinetic_u3": "Telekinetic Utility III",
   "item.minebound.magic_telekinetic_u4": "Telekinetic Utility IV",
 
+  "item.minebound.magic.magic_telekinetic_o1": "Telekinetic Offensive I",
```

Comment:
Do we want to collect the dummy spells into their own category instead of having them located so close to the core spells?